lib/prodder/pg.rb

```diff
@@ -62,9 +62,10 @@ def dump_settings(db_name, filename)
         raise PGDumpError.new(err) if !success
         File.open(filename, 'w') do |f|
           out.each_line do |setting|
-            unless setting.gsub(/\s+/, '').empty?
+            setting.gsub!(/\s+/, '')
```

Comment:
It's cool to leave this as is, but the `+` in `/\s+/` is unnecessary because `gsub!` will replace all instances.